src/state/slices/swapSlice/types.ts

```diff
@@ -0,0 +1,7 @@
+import type { Swap } from '@shapeshiftoss/swapper'
+
+export type SwapState = {
+  swaps: Swap[]
+}
+
+export type UpdateSwapPayload = Partial<Swap> & { id: string }
```

Comment:
q: why is this partial?

Reply:
Because updating a swap can be partial eg. txHash or currentStep, but happy to spread the already existing swap and keep things even more "simpler"